src/core/textrenderer/qgstextshadowsettings.cpp

```diff
@@ -43,6 +43,33 @@ QgsTextShadowSettings::~QgsTextShadowSettings() //NOLINT
 
 }
 
+bool QgsTextShadowSettings::operator==( const QgsTextShadowSettings &other ) const
+{
+  if ( d->enabled != other.enabled()
+       || d->shadowUnder != other.shadowPlacement()
+       || d->offsetAngle != other.offsetAngle()
+       || d->offsetDist != other.offsetDistance()
+       || d->offsetUnits != other.offsetUnit()
+       || d->offsetMapUnitScale != other.offsetMapUnitScale()
+       || d->offsetGlobal != other.offsetGlobal()
+       || d->radius != other.blurRadius()
```

Comment:
radius == blurRadius? I'm just asking to make sure, we did change quite a few variable / function names during the 3.X cycle.

Reply:
Yep -- it's just the internal member name which wasn't updated